modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb

```diff
@@ -10,6 +10,20 @@ def initialize(data)
       @data = data
     end
 
+    def as_payload
+      {
+        remarks: data['remarks'],
+        otherConditions: data['other_conditions'],
+        livingSituation: living_situation_payload,
+        previousHiApplication: previous_hi_application_payload,
+        previousSahApplication: previous_sah_application_payload,
+        veteran: veteran_payload,
+        statementOfTruthSignature: data['statement_of_truth_signature'],
+        statementOfTruthCertified: data['statement_of_truth_certified'],
+        formNumber: data['form_number']
+      }
+    end
```

Comment:
Something to consider, I think you can transform the snake_case keys into camelCase relatively painlessly like so:

```ruby
data.transform_keys { |key| key.to_s.camelize(:lower) }
```

It might be worth a shot either in this PR or in a future refactor PR.

Reply:
I toyed around with this a little and recursion would indeed have been necessary. I think I'll stick with what I have since it does let us be more flexible in case the API changes what they need. It's definitely circuitous and verbose but it's at least explicit. 🤷